elementary/files
# Fix overlaybar enter

When pointer enters the overlay statusbar, it should move to the other side of the window.

In master, the statusbar disappears.

In this branch the statusbar appears on the other side and continues any ongoing operation (deep count or resolution determination).

Because it is not possible to distinguish directly between the pointer entering the overlay and the pointer leaving the view window entirely, the branch works by delaying the cancellation and hiding of the statusbar briefly.  If the update event with null parameter is due to entering the overlay then it will be followed almost immediately by another update event with the same file as the previous non-null event.  In this case, the null event is ignored.  If the pointer leaves the window there is no following event and the statusbar hides.
  
TO TEST:
* Confirm the issue is fixed.
* Check that there are no regressions from the expected behaviour when the mouse is hovered over file items in view.
* Check that there are no regressions from the expected behaviour when the mouse is hovered over other parts of the app . Only code for view and overlay are changed.
